cli/cli.go

```diff
@@ -84,18 +80,17 @@ List of global options:       {{.Parent.Name}} help -g
 `
 
 	GLOBAL_OPTIONS_TEMPLATE = `
-
 Global Options:
 {{range .Flags }}       {{flagAligner .FlagStringPrefix}} {{.ShortDesc}}
 {{end}}
 
 `
 
-//{{range .Flags}}{{if len .Short}}-{{.Short}},{{end}}--{{.Long}}{{if isOption .}}   {{upper .Long}}{{end}}       {{.Description}}{{end}}
+// {{range .Flags}}{{if len .Short}}-{{.Short}},{{end}}--{{.Long}}{{if isOption .}}   {{upper .Long}}{{end}}       {{.Description}}{{end}}
```

Comment:
All this reformatting is good (or most of it), but it would be nice if it were in a separate commit, so we could better see what actually changed (namely the `bringUp` function).

Reply:
(Same remark for the "chore: cleanup" commit.)

Reply:
(Found a way to deactivate format on save enforced by my vscode Go extension, on the way to ditch it)